coder_sniffer/Drupal/ruleset.xml

```diff
@@ -297,4 +297,7 @@
  <exclude-pattern>*/\.svn/*</exclude-pattern>
  <exclude-pattern>*/\.hg/*</exclude-pattern>
  <exclude-pattern>*/\.bzr/*</exclude-pattern>
+ 
+ <!-- Ignore min.css files-->
+ <exclude-pattern>*.min.css</exclude-pattern>
```

Comment:
since dots are special characters in regex we should escape them with backslashes, right? We should also add a $ at the end.

Reply:
Yes, totally makes sense, thanks!